libs/components/src/lib/listbox/listbox.ts

```diff
@@ -22,25 +23,29 @@ export class Listbox extends FoundationListboxElement {
 	 * @remarks
 	 * HTML Attribute: appearance
 	 */
-	@attr appearance?: LisboxAppearance;
+	@attr appearance?: ListboxAppearance;
 
-	override slottedOptionsChanged(prev: Element[] | undefined, next: Element[]) {
-		super.slottedOptionsChanged(prev, next);
-		this.#disableSlottedChildren();
-	}
+	/**
+	 * To save the state of children options before going disabled.
+	 */
+	private _optionsState: boolean[] = [];
 
-	override attributeChangedCallback(name: string, oldValue: string, newValue: string): void {
-		super.attributeChangedCallback(name, oldValue, newValue);
-		if (name === 'disabled') {
-			this.#disableSlottedChildren();
+	/**
+	 * Saves/restore children state when the `disabled` property changes.
+	 *
+	 * @param _ - The previous disabled value
+	 * @param becomesDisabled - The next disabled value
+	 * @internal
+	 */
+	disabledChanged(_: boolean, becomesDisabled: boolean): void {
```

Comment:
align to lifecycles common convention? (`prev` / `next`)
```suggestion
	disabledChanged(_: boolean, next: boolean): void {
```